tembo-operator/src/dedicated_networking.rs

```diff
@@ -399,12 +399,6 @@ async fn reconcile_dedicated_networking_service(
         "cnpg.io/cluster".to_string(),
         serde_json::Value::String(cdb_name.to_string()),
     );
-    if is_public {
```

Comment:
Is this not needed anymore?

Reply:
I don't think so. We have "service.beta.kubernetes.io/aws-load-balancer-internal: "false" for that

Reply:
Ok Thanks.